### PR TITLE
Fixed the clang compiler tool so it uses the correct target triple.

### PR DESCRIPTION
**Pull request by zoejbare**

### CODE REVIEW

File: csbuild/tools/cpp_compilers/clang_cpp_compiler.py [hunk 1]
```diff
@@ -68,9 +76,15 @@ def _getArchitectureArgs(self, project):
 	####################################################################################################################
 
 	def _getArchTarget(self, project):
+		if not self._targetSuffix:
+			return None
+
 		# When necessary fill in the architecture name with something clang expects.
-		architecture = {
+		targetPrefix = {
```
Comment (ShadauxCat) [inline csbuild/tools/cpp_compilers/clang_cpp_compiler.py:83]: Commenting here just because I don't like you.

Dr. Evazan doesn't like you either.

**Review by ShadauxCat — APPROVED** [review]

File: csbuild/tools/cpp_compilers/clang_cpp_compiler.py (hunk 1 above)
Comment (zoejbare) [reply csbuild/tools/cpp_compilers/clang_cpp_compiler.py:83]: I'll be careful.

Comment (ShadauxCat) [reply csbuild/tools/cpp_compilers/clang_cpp_compiler.py:83]: You'll be dead!

Comment (zoejbare) [reply csbuild/tools/cpp_compilers/clang_cpp_compiler.py:83]: This little one (referring to myself) isn't worth the effort.  Come, let me get you something.

Comment (ShadauxCat) [reply csbuild/tools/cpp_compilers/clang_cpp_compiler.py:83]: You don't want to get me death sticks. You want to go home and rethink your life.

Comment (zoejbare) [reply csbuild/tools/cpp_compilers/clang_cpp_compiler.py:83]: I want to go home and rethink my life...

**Pull request merged**